app/about/the-team/page.tsx

```diff
@@ -1,4 +1,5 @@
 "use client";
+import { TeamMember, TeamMemberGroup } from "@/gql/graphql";
```

Comment:
Where we import the types from!

Reply:
My one word of caution here would be that you're importing the type that is derived from the **graphql schema**, not the type that's derived from the **query**. This means that if we add a new field to `TeamMember` on the backend but don't request it on the front end, the build will break because `groupedTeamMembers['teamMembers']` would no longer be assignable to `TeamMember[]` since that type has more fields now. The only reason this typecast works is because this query is requesting every single field on `TeamMember`, if that stopped being true, this would stop working.

Anyways, I'm not saying this is something we need to fix right now but if you want to fix this your two options are:
- Define a new `TeamMember` type that is `type TeamMember = groupedTeamMembersQuery['teamMembers'][number]` (or something like that). This will be based on your query type and we'll be a-ok if you add new fields on the backend
- Redefine the `TeamMember` type including only fields that are used by this component.

Both work, this also works for now just a quick warning hahahaha